Tools/Scripts/webkitpy/layout_tests/models/test_expectations.py

```diff
@@ -715,6 +720,10 @@ def add_expectation_line(self, expectation_line, in_skipped=False):
             return
 
         for test in expectation_line.matching_tests:
+            if test not in self._test_to_expectation_lines:
+                self._test_to_expectation_lines[test] = []
+            self._test_to_expectation_lines[test].append(expectation_line)
```

Comment:
Performance is unlikely to be too critical here, but we can avoid having to access the item on the dict multiple times:

```suggestion
            self._test_to_expectation_lines.setdefault(test, []).append(expectation_line)
```

Reply:
Didn't know about `setdefault`, neat, thanks!